File: java/src/main/java/giis/selema/framework/junit4/Asserts.java
package giis.selema.framework.junit4;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;

/**
 * Asserts for compatibility in translation from JUnit4 to NUnit3
 */
public class Asserts {
	private Asserts() {
		throw new IllegalStateException("Utility class");
	}
	
	public static void assertAreEqual(String expected, String actual, String message) {
		assertEquals(message, expected, actual);		
	}

	public static void assertIsTrue(boolean condition, String message) {
		assertTrue(message, condition);		
	}
}

File: net/Selema/Giis.Selema.Framework.Nunit3/Asserts.cs
using NUnit.Framework;

namespace Giis.Selema.Framework.Nunit3
{
    /// <summary>
    /// Asserts for compatibility in translation from JUnit4 to NUnit3
    /// </summary>
    public class Asserts
    {
        public static void AssertAreEqual(string expected, string actual, string message)
        {
            Assert.AreEqual(expected, actual, message);
        }
        public static void AssertContains(string value, string substring)
        {
            StringAssert.Contains(substring, value);
        }
        public static void AssertIsTrue(bool condition, string message)
        {
            Assert.IsTrue(condition, message);
        }
    }
}
